test_project/dp_settings.py

```diff
@@ -149,3 +147,9 @@
 # File to save revoked tasks across workers restart
 CELERY_WORKER_STATE_DB = os.path.join(TMP_DIR, 'celery_states')
 CELERY_BEAT_SCHEDULE_FILENAME = os.path.join(TMP_DIR, 'celerybeat_schedule')
+
+# just in testing
+CELERY_ALWAYS_EAGER = True
+CELERY_TASK_EAGER_PROPOGATES=True
+CELERY_RESULT_BACKEND = 'db+sqlite:///results.db'
+CELERY_BROKER_URL = 'memory://localhost//'
```

Comment:
Please create a test settings file for this, so that it doesn't affect the development environment. You would have to set the `DJANGO_SETTINGS_MODULE` env variable for the tests.

Reply:
Any update on this?.